frontend/src/App.tsx

```diff
@@ -1,18 +1,18 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState } from "react";
+import reactLogo from "./assets/react.svg";
+import viteLogo from "../public/vite.svg";
+import "./App.css";
```

Comment:
Is this meant to be single quote instead?

Reply:
Guess who forgot to run prettier after rebasing? Should be fixed now.